profiles/base.yaml

```diff
@@ -2146,6 +2148,7 @@ tarantula_noun: biomechanical tarantula
 tarantula_startup_delay: 15
 tarantula_skip_alternate: false
 tarantula_no_use_scripts:
+  - burgle
```

Comment:
Not completely sure what's going on here but GitHub is identifying this in the diff and it looks unrelated to this PR. Also appears to already exist in `base.yaml`.

Reply:
I must have put in before it was merged. I can fix with a new push.